cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2024-12-13 23:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(null=True, upload_to="movie_image_path"),

```

Comment:
Ensure that the `ImageField` added here is not accessible via POST requests to `api/cinema/movies/`, as per the task requirements. This field should only be used via the `/upload-image/` endpoint.